packages/web-components/src/divider/divider.options.ts

```diff
@@ -0,0 +1,49 @@
+import { DividerRole, ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Fast Foundation DividerRole property
+ * @public
+ */
+export { DividerRole };
+
+/**
+ * Align content within divider
+ * @public
+ */
+export const DividerAlignContent = {
+  center: 'center',
+  start: 'start',
+  end: 'end',
+} as const;
+
+/**
+ * The types for DividerAlignContent
+ * @public
+ */
+export type DividerAlignContent = ValuesOf<typeof DividerAlignContent>;
+
+/**
+ * DividerAppearance - divider color defined by a design token alias.
+ * @public
+ */
+export const DividerAppearance = {
+  strong: 'strong',
+  brand: 'brand',
+  subtle: 'subtle',
+  default: 'default',
+} as const;
+
+/**
+ * The types for Appearance
+ * @public
+ */
+export type DividerAppearance = ValuesOf<typeof DividerAppearance>;
+
+/**
+ * Inset - display block or inline
+ * @public
+ */
+export const Inset = {
+  true: true,
+  false: false,
+} as const;
```

Comment:
I don't think we need a const object for boolean attributes, this can be removed

Reply:
Remove the whole thing?

Reply:
Yes, const objects are great for examples such as above where you have unique values - in this case, the type is "boolean" rather than `DividerInset`, so it's either `true` or `false` in JS. I don't think const objects offer value in this case, they simply add bytes.